centralsettlement/configs/default.json

```diff
@@ -7,5 +7,42 @@
     "HUB_PARTICIPANT": {
         "ID": 1,
         "NAME": "Hub"
-    }
+    },
+    "SIDECAR": {
```

Comment:
I don’t believe you need sidecar config here

Reply:
Fixed